applications/luci-app-aria2/htdocs/luci-static/resources/view/aria2/files.js

```diff
@@ -0,0 +1,54 @@
+'use strict';
+'require fs';
+'require uci';
+'require view';
+
+return view.extend({
+	load: function() {
+		return uci.load('aria2').then(function() {
+			var config_dir = uci.get_first('aria2', 'aria2', 'config_dir') || '/var/etc/aria2';
+			var config_file = '%s/aria2.conf.main'.format(config_dir);
+			var session_file = '%s/aria2.session.main'.format(config_dir);
+
+			return Promise.all([
+				fs.exec_direct('/bin/cat', [ config_file ]).then(function(res) { return { 'file': config_file, 'rows': res.trim().split(/\n/).length, 'content': res.trim() } }),
```

Comment:
This is a matter of taste, but I think the lines are too long. Can you please reformat them?

Reply:
Is this OK now?